examples/repl.rs

```diff
@@ -138,7 +138,8 @@ mod example {
                 if let Some(pid) = parts.next() {
                     let pid = nix::unistd::Pid::from_raw(pid.parse()?);
                     println!("Attaching to process {}", pid);
-                    let (remote, status) = LinuxTarget::attach(pid)?;
+                    let (remote, status) =
+                        LinuxTarget::attach(pid, AttachOptions { kill_on_exit: true })?;
```

Comment:
Can this be falss? gdb and lldb detach on exit when they haven't spawned the process themself.